gammapy/modeling/models/spectral.py

```diff
@@ -289,15 +289,15 @@ def plot(
             from astropy import units as u
 
             pwl = ExpCutoffPowerLawSpectralModel()
-            ax = pwl.plot(energy_range=(0.1, 100) * u.TeV)
+            ax = pwl.plot(energy_bounds=(0.1, 100) * u.TeV)
             ax.set_yscale('linear')
 
         Parameters
         ----------
         ax : `~matplotlib.axes.Axes`, optional
             Axis
-        energy_range : `~astropy.units.Quantity`
-            Plot range
+        energy_bounds : `~astropy.units.Quantity`
```

Comment:
Mention `energy_bounds` is forward to `MapAxis.from_energy_bounds()` here.

Reply:
Done.